src/XamlStyler/DocumentManipulation/NodeCollection.cs

```diff
@@ -1,9 +1,10 @@
-// � Xavalon. All rights reserved.
+// � Xavalon. All rights reserved.
```

Comment:
I hope this is just GitHub doing strange things.

Reply:
There was some odd characters in my most recent check-in too... not sure what is going on.

Reply:
Seems to be an encoding issue with some files that are not in UTF-8 it seems. Fixing the two files touched in this PR in the next commit.